### ISSUE TITLE
Modern browsers as default

### ISSUE THREAD

**Issue by theel0ja** [majodev/google-webfonts-helper#60]
https://caniuse.com/#search=woff

Well, nearly all browsers support (94.53% support, Opera Mini and old legacy browsers don't) support WOFF and/or WOFF2, so I think that "Modern Browsers" should be default.

**Comment by steebchen** [comment]
Agree. It doesn't really matter though, because browsers only load the font file they use anyway (for example, latest Chrome/FF loads woff2).

**Comment by majodev** [comment]
Closed via https://github.com/majodev/google-webfonts-helper/pull/173 (via https://github.com/majodev/google-webfonts-helper/issues/162).
